ojd_daps_skills/pipeline/skill_ner/combine_labels.py

```diff
@@ -112,11 +118,90 @@ def combine_results(labelled_data_s3_folders, keep_id_dict, metadata_jobids):
                 job_labels[job_id] = {
                     "text": job_advert_labels["task"]["data"]["text"],
                     "labels": job_advert_labels["result"],
+                    "type": "label-studio",
                 }
 
     return job_labels
 
 
+def load_format_prodigy(prodigy_labelled_data_s3_folders):
+    """
+    Load all prodigy labels
+    Since these were labelled in 5 sentence chunks, then sort them into a nested dict
+    with the job advert id and the sentence chunk number
+
+    """
+    s3 = get_s3_resource()
+    prodigy_data_chunks = defaultdict(dict)
+    for prodigy_labelled_data_s3_folder in prodigy_labelled_data_s3_folders:
```

Comment:
is there a reason why we're looping as it looks like we only have one filename? Or am i miss understanding?

Reply:
no you are right! I was copying the structure of label-studio where we had multiple files without thinking. Will adapt